shared/js/bluetooth_helper.js

```diff
@@ -141,6 +144,7 @@ var BluetoothHelper = function() {
     } else {
       _bluetooth.addEventListener('enabled', _getAdapter);
       _bluetooth.addEventListener('adapteradded', _getAdapter);
+      _bluetooth.addEventListener('disabled', _resetAdapter);
```

Comment:
I tried to add test for it, but seems we can't get properties such as `assert.isFalse(subject._isReady)` or `subject.resetAdapter.called` for validation.